### PR TITLE
Private Event Test

### PR DESCRIPTION
**Pull request by didymus707**
Added unit and integration test for the project

### CODE REVIEW

File: app/views/partials/_invite_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  <%#= form_for(@event_attendance) do |f| %>
+  <!--<div class="row">
+    <%#= f.label :attended_event_id %>
+    <%#= f.collection_select :attended_event_id, Event.all, :id, :description, prompt: true %>
+  </div>
+
+  <div class="row">
+    <%#= f.label :event_attendee_id %>
+    <%#= f.collection_select :event_attendee_id, User.where.not(id: current_user.id), :id, :name, prompt: true %>
+  </div>-->
```
Comment (wathigo) [inline app/views/partials/_invite_form.html.erb]: **REQUIRED**
Let's delete the commented code

**Review by wathigo — CHANGES_REQUESTED** [review]
### Changes Required
Great work adding unit and integration tests for the application.
Also great work on the styles!

**REQUIRED**
*  Event managment features should have integration tests (Rspec + Capybara) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/integration_tests_requirement.png) from Pathwright).

**After implementing the changes, submit another request for review.**

**Happy Hacking :computer: 
Simon Wathigo**

File: spec/features/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    }
+  end
+
+  def sign_up(new_user)
+    within '.column' do
+      fill_in 'user[name]', with: new_user[name]
+      fill_in 'user[email]', with: new_user[email]
+
+      click_button 'Sign up'
+    end
```
Comment (maelfosso) [inline spec/features/user_spec.rb]: What next? You must expect the user to be created into the database and after clicking on the sign up button, a new page appears, routing has been done. you should expect all those things.

File: spec/features/user_spec.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  end
+
+  def sign_in(user)
+    vist new_sessions_path
+    within '.column' do
+      fill_in 'user_name', with: user.name
+
+      click_button 'Log in'
+    end
+  end
```
Comment (maelfosso) [inline spec/features/user_spec.rb]: Same thing here. What is the sign of a successful log in process.

File: spec/features/user_spec.rb [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+  def manage_events(user, event)
+    visit attend_path
+
+    within '.column' do
+      fill_in 'event_attendance_id', with: event[description]
+      fill_in 'event_attendee_id', with: user.name
+    end
+  end
```
Comment (maelfosso) [inline spec/features/user_spec.rb]: Same problem!!!

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Thank you for applying the suggested changes.
However, it's not sufficient.

1. You **must check whether or not the test is OK**. Do that by checking the expected result with the really one

2. Write test that also failed.

3- Read the comments below

**CHANGES REQUIRED** :red_circle: 

Kindly make the changes and submit a new code review request
Thanks

Mael FOSSO

**Comment by uryelah** [comment]
Hi **Didymus Orotayo**, I'm Sarah your reviewer for this PR 👩‍🚀 

Great job implementing the suggestions from the last review!

-------

## **Your project is approved! 🎊 🥅 💛**

**You may close this Pull Request and merge the changes**


---------

    
    
**Keep being awesome devs and good luck with your next project!**

---------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**